source/broadband/webpa_notification.c

```diff
@@ -935,8 +1036,9 @@ static void handleNotificationEvents()
 			WAL_FREE(message);
 		}
 		else
-		{		
-			WalPrint("handleNotificationEvents : Before pthread cond wait in consumer thread\n");   
+		{	
+			g_syncNotifyInProgress = 0;	
+			WalInfo("handleNotificationEvents : Before pthread cond wait in consumer thread\n");   
```

Comment:
Move this as WalPrint . And then add one new Info log saying "g_syncNotifyInProgress is set to 0"